temporalcli/commandsgen/commands.yml

```
@@ -2886,6 +2886,9 @@ commands:
       - name: limit
         type: int
         description: Maximum number of Workflow Executions to display.
+      - name: pagesize
```

To be consistent with how we name other flags, this should be:

```suggestion
      - name: page-size
```